conans/client/command.py

```diff
@@ -895,6 +895,8 @@ def run(self, *args):
                 self._user_io.out.error("Exiting with code: %d" % exc.code)
             errors = exc.code
         except ConanException as exc:
+            import traceback
+            print(traceback.format_exc())
```

Comment:
Intended or just debugging lines?

Reply:
Debugging, have to remove

Reply:
Yes, the implementation (or des-implementation) looks good. Let's fire the dev-requires!